includes/apple-exporter/class-theme.php

```diff
@@ -730,6 +730,37 @@ private static function initialize_options() {
 				'label'       => __( 'Body tracking', 'apple-news' ),
 				'type'        => 'integer',
 			],
+			'button_background_color'            => [
+				'default' => '#ffffff',
+				'label'   => __( 'Background color', 'apple-news' ),
+				'type'    => 'color',
+			],
+			'button_border_color'                => [
+				'default' => '#000000',
+				'label'   => __( 'Border color', 'apple-news' ),
+				'type'    => 'color',
+			],
+			'button_border_radius'               => [
+				'default' => 18,
+				'label'   => __( 'Border radius', 'apple-news' ),
+				'type'    => 'integer',
+			],
+			'button_border_width'                => [
+				'default' => 1,
+				'label'   => __( 'Border width', 'apple-news' ),
+				'type'    => 'integer',
+			],
```

Comment:
Need to identify these properties as belonging to buttons:
```suggestion
			'button_background_color'            => [
				'default' => '#ffffff',
				'label'   => __( 'Button background color', 'apple-news' ),
				'type'    => 'color',
			],
			'button_border_color'                => [
				'default' => '#000000',
				'label'   => __( 'Button border color', 'apple-news' ),
				'type'    => 'color',
			],
			'button_border_radius'               => [
				'default' => 18,
				'label'   => __( 'Button border radius', 'apple-news' ),
				'type'    => 'integer',
			],
			'button_border_width'                => [
				'default' => 1,
				'label'   => __( 'Button border width', 'apple-news' ),
				'type'    => 'integer',
			],
```